client/web/src/components/views/readonly-client-view.tsx

```diff
@@ -51,18 +70,22 @@ export default function ReadonlyClientView({
               <div className="text-sm leading-tight">{data.IP}</div>
             </div>
           </div>
-          {auth?.authNeeded == AuthType.tailscale && (
+          {auth?.authNeeded == AuthType.tailscale ? (
+            <button className="button button-blue ml-6" onClick={newSession}>
+              Access
+            </button>
+          ) : window.location.hostname != data.IP ? (
+            // TODO: check connectivity to tailscale IP
             <button
               className="button button-blue ml-6"
               onClick={() => {
-                sessions
-                  .new()
-                  .then((url) => window.open(url, "_blank"))
-                  .then(() => sessions.wait())
+                window.location.href = `http://${data.IP}:5252/?checkmode=1`
               }}
             >
-              Access
+              Manage
             </button>
+          ) : (
+            <></>
```

Comment:
Instead of this, the above should just be
```
) : window.location.hostname != data.IP && (
```
The stuff after `&&` will only render if the condition is satisfied, so we don't have to have this extra bit down here.

Reply:
ah, that's the syntax I was looking for :)